features/lwipstack/mbed_lib.json

```diff
@@ -77,10 +77,26 @@
             "help": "Number of simultaneously queued TCP segments. Current default (used if null here) is set to 16 in opt.h, unless overridden by target Ethernet drivers.",
             "value": null
         },
+        "memp-num-tcpip-msg-inpkt": {
+            "help": "Number of simultaneously queued TCP messages that are received. Current default (used if null here) is set to 8 in opt.h, unless overridden by target Ethernet drivers.",
+            "value": null
+        },
         "tcp-mss": {
             "help": "TCP Maximum segment size. Current default (used if null here) is set to 536 in opt.h, unless overridden by target Ethernet drivers.",
             "value": null
         },
+        "tcpip-mbox-size": {
+            "help": "TCPIP mailbox size. Current default (used if null here) is set to 8 in lwipopts.h, unless overridden by target Ethernet drivers.",
```

Comment:
These two settings - tcpip-mbox-size, default-tcp-recvmbox-size have never had `#ifndef` on them, so the "overridden by target Ethernet drivers" doesn't apply.

The `null` stuff only applies when we want to defer to either some target defines (basically a pre-JSON legacy mechanism), or we want to defer to lwIP's own default. Here, lwIP has no default and we have to specify.

Probably simplest to make them straight options with no defaults

    "tcpip-mbox-size": {
            "help": "TCPIP mailbox size.",
            "value": 8
     },

    #define TCPIP_MBOX_SIZE             MBED_CONF_LWIP_TCPIP_MBOX_SIZE

Reply:
Ok I will specify this to default 8.

Reply:
All the blurb about "defaults" in the help can go.

Reply:
Done